experimental/Schemes/SpaceGerms.jl

```diff
@@ -63,37 +201,120 @@ end
     return Spec(R)
 end
 
-function point(X::SpaceGerm{<:Any, <:Any, <:GermAtClosedPoint})
+@doc raw"""
+    point(X::AbsSpaceGerm)
```

Comment:
Please adjust the docstring to the new method's signature below.

Reply:
done